assets/dev/js/editor/regions/responsive-bar/view.js

```diff
@@ -118,8 +120,8 @@ export default class View extends Marionette.ItemView {
 
 		const size = elementor.channels.responsivePreview.request( 'size' );
 
-		this.ui.sizeInputWidth.val( size.width );
-		this.ui.sizeInputHeight.val( size.height );
+		this.ui.sizeInputWidth.val( Math.round( size.width ) );
+		this.ui.sizeInputHeight.val( Math.round( size.height ) );
```

Comment:
In which case the value is not an integer?

Reply:
In this case:
https://www.loom.com/share/46f0f5cc464846bc95680084ad0a902f